bitcoin/CHANGELOG.md

```diff
@@ -68,7 +68,7 @@ For changes to our dependencies included in this release see:
 - transaction: Remove `Default` implementations [#3386](https://github.com/rust-bitcoin/rust-bitcoin/pull/3386)
 - Add `FeeRate` addition and subtraction traits [#3381](https://github.com/rust-bitcoin/rust-bitcoin/pull/3381)
 - Add `Xpriv::to_xpub` and improve related method names [#3358](https://github.com/rust-bitcoin/rust-bitcoin/pull/3358)
-- Support `impl AsRef<[#u8]>` in `signed_msg_hash` [3357](https://github.com/rust-bitcoin/rust-bitcoin/pull/u8)
+- Support `impl AsRef<[#u8]>` in `signed_msg_hash` [#3357](https://github.com/rust-bitcoin/rust-bitcoin/pull/3357)
```

Comment:
This looks suspiciously like I used an emacs macro to insert the `#` and didn't look close enough.

```suggestion
- Support `impl AsRef<[u8]>` in `signed_msg_hash` [#3357](https://github.com/rust-bitcoin/rust-bitcoin/pull/3357)
```

Reply:
Emacs is an OS that lacks a good editor, maybe try nvim. ;)